lib/stores/MemcachedCacheStore.js

```diff
@@ -32,7 +41,7 @@ module.exports = class MemcachedCacheStore extends CacheStore {
 
   _multiGet(key) {
     return new Promise((res, rej) => {
-      this._queuedKeys.push(key);
+      this._queuedKeys.push(this._toCacheKey(key));
```

Comment:
should we normalize the cache key here or just in the public `get` method of this class?